packages/react-sdk/docusaurus/docs/React/02-tutorials/02-audio-room.mdx

```diff
@@ -3,4 +3,543 @@ title: Audio Room Tutorial
+
+To do this we need to query the backend for a list of the calls and display each of the rooms. Again, we'll not focus on the UI
+part since that's different for each use case.
+
+So, let's assume we have a component that stores the rooms in a `rooms` array, so let's define that at the root of the component:
+
+```
+const [rooms, setRooms] = useState<AudioRoom[]>([]);
+```
+
+Next, querying is fairly easy. Here is the code and we'll go through it afterwards:
+
+```tsx
+const client = useStreamVideoClient();
+useEffect(() => {
+  console.log('Loading calls');
+  client
+    ?.queryCalls({
+      filter_conditions: { audioRoomCall: true },
+      sort: [],
+      watch: true,
+    })
+    .then((result) => {
+      console.log('Querying calls successful.');
+      setRooms(result.calls);
+    })
+    .catch((err) => {
+      console.log('Querying calls failed.');
+      console.error(err);
+    });
+}, [client, setRooms]);
+```
+
+We need a reference to the `client` which we conveniently can get with the `useStreamVideoClient` hook.
+Then, we call the `.queryCalls` function in a `useEffect` hook. We can specify three things there:
+
+1. `filter_conditions`: we want to only have calls with the `audioRoomCall` property set to true (which we specified earlier)
+2. `sort`: we _could_ sort the calls we receive by some logic we want, for simplicity reasons we don't do this here
+3. `watch`: this simple, but powerful property gives us the power to listen to real-time changes; whenever a call changes (for example going live) we'll get notified, and our rooms list updates
+
+With that, we're ready to show the list of rooms. You can define that as you like, our example can be found [here](TODO: provider link to `RoomOverview` file).
+
+### Joining and leaving a room
+
+To join a room, we need the `Call` variable that we attached to the room. With that, we can use the
+simple `.join()` and `.leave()` functions on it.
+
+Let's assume we put the call into a new component and use a `useEffect` hook to call `.join()` when it gets mounted:
+
+```tsx
+const joinCall = useCallback(async (call: Call) => {
+  await call.join();
+}, []);
+
+useEffect(() => {
+  joinCall();
+
+  return () => {
+    call.leave();
+  };
+});
+```
+
+This will tell the SDK to join the room when the component is rendered and with the callback, we return in `useEffect` it will also leave calls when it's unmounted.
+
+Next, to show the room and provide audio output and input we need to wrap the component that does that with two providers:
+
+1. `MediaDevicesProvider`: this one is needed to give access to audio and video devices; it has two parameters, `initialAudioEnabled` and `initialVideoEnabled` which do exactly what they describe.
+2. `StreamCallProvider`: it is initialized with a `Call` and provides us with a lot of useful and convenient hooks to have easy access to the necessary call parameters
+
```

Comment:
There were some changes around this, you can use the `StreamCall` component here: https://github.com/GetStream/stream-video-js/blob/main/packages/react-sdk/src/core/components/StreamCall/StreamCall.tsx